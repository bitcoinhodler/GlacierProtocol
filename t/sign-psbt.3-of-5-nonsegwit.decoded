{
  "txid": "69751d8743c5b6d9806bb02def9320ba11b2fc396e75a8c50d975608e4caccea",
  "hash": "69751d8743c5b6d9806bb02def9320ba11b2fc396e75a8c50d975608e4caccea",
  "version": 2,
  "size": 479,
  "vsize": 479,
  "weight": 1916,
  "locktime": 0,
  "vin": [
    {
      "txid": "8d71580c7f378636cbd8ebd3e0bb01f72ad3de642d064eaef9bae0eec4ac561d",
      "vout": 2,
      "scriptSig": {
        "asm": "0 304402205b215649efee3668ea4df5b38023f4232c9db9077588ef20b6d242858c8523f10220557bfe5b4345b9ec9773f7ce1729dfbdddc7a9987964c8ec8f7b932b56056239[ALL] 3044022044f96125d54319e92e876965756b4c8188a979ea86424bef9c136c52466cf2a80220723179ff9e394f5ac354c8d2d109d302defe904a8c15a5a351b80e183ea68cfe[ALL] 304402203e55ded80ee250564b7a01326fb1e87694c0f59556fd55a0e15885a22cc242f30220382746d2220cebaae3aefdabe19eed3adae8ec8c01c559969b3260a2956d55de[ALL] 53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155ae",
        "hex": "0047304402205b215649efee3668ea4df5b38023f4232c9db9077588ef20b6d242858c8523f10220557bfe5b4345b9ec9773f7ce1729dfbdddc7a9987964c8ec8f7b932b5605623901473044022044f96125d54319e92e876965756b4c8188a979ea86424bef9c136c52466cf2a80220723179ff9e394f5ac354c8d2d109d302defe904a8c15a5a351b80e183ea68cfe0147304402203e55ded80ee250564b7a01326fb1e87694c0f59556fd55a0e15885a22cc242f30220382746d2220cebaae3aefdabe19eed3adae8ec8c01c559969b3260a2956d55de014cad53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155ae"
      },
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.19995210,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 b6c7733e23a271e57ca16e5cd885174917968db9 OP_EQUALVERIFY OP_CHECKSIG",
        "desc": "addr(mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99)#fxf0ktmw",
        "hex": "76a914b6c7733e23a271e57ca16e5cd885174917968db988ac",
        "address": "mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99",
        "type": "pubkeyhash"
      }
    }
  ]
}
