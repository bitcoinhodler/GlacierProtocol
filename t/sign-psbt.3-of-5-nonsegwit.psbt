cHNidP8BAFUCAAAAAZi8RZ7Uy0KwXQLefZOvdzTxunsesOldFrrJ1IyqW6NzAgAAAAD/////AUoaMQEAAAAAGXapFLbHcz4jonHlfKFuXNiFF0kXlo25iKwAAAAAAAEA/T8BAgAAAAH6WU9oe79CjaFIn4OAK3YXhOrbqZJvTr+fwxBG8i/FUQEAAABqRzBEAiB+MV9QGcXRYRfME/aLiBIWhY8Diz+WTtA8A15fWHx7VAIgE7DGmHF1DRsUKA2Wlz/gABcgEjyRtRhkGGkjlvEC8bQBIQMPTMGA5Od2IsW0G2895rCQhqWcFQOWY0AcjurEsZ8pV/////8FAC0xAQAAAAAXqRQirZZC1y63k5fHhx6cr76agKCrfoeQQJcAAAAAABl2qRQciCY7X9EUVYXfQpSb3wFzj6YWpIisAC0xAQAAAAAXqRSXwv/c38Izoyh1G0ake3gbHuybLYcALTEBAAAAABepFGa45TJV9p0DmJVNH4BPqaCpO/85hwAtMQEAAAAAF6kUBPw6eRbYcwxzzrA+NcAvZQWXDAqHAAAAAAEErVMhAyCElgfu0mXAn1HNoOlyQmVYJ2OEhwZRJQqk6GOMM0ztIQJ2bHBHKoo951HRJu3ayENjOPhEexlntao7EqK5pucS6SEDkIDdxEjn/kmFZ2p9LWVn38L+u4IHpvKblzcjP0w7RNYhA/otLAdlOvsKczQO6eaxD/DUYkyTsBEn6aae++1IRo4BIQKWabedfgXyDKeqQY7cg52pBEZyU8nkcZTLg+ae5BgxUVWuAAA=
