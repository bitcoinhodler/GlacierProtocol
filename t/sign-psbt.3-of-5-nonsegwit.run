#!/bin/bash
set -e

$GLACIERSCRIPT --regtest=$1 sign-psbt << INPUT 2>&1
y
y
y
y
y
y
../../t/sign-psbt.3-of-5-nonsegwit.psbt
y
3
cQy39MvkMKeaEvG1wXejdtEvAdU381u9EBBmcKUJWbfn6QL82YvR
cUv81BVz1JWuA9uwECPvfsoyA1TqmFJBJaQUPYrxkdf44zQSiTZY
cVV1MBAgdAnrX8bjQWSJcgZAbVtprR8kgdfez858ejvXs3qZWqn2
y
INPUT
