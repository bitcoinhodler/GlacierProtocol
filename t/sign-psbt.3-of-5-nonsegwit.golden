Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.20000000 BTC in unspent inputs from cold storage address 2N75fd3tacsz3g25Eak6XnP7wizd8scVnGe
0.19995210 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
Fee amount: 0.00004790
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 3 required to complete transaction)?
#: Key #1: Key #2: Key #3: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.20000000 BTC in unspent inputs from cold storage address 2N75fd3tacsz3g25Eak6XnP7wizd8scVnGe
0.19995210 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
Fee amount: 0.00004790

Signing with private keys: 
cQy39MvkMKeaEvG1wXejdtEvAdU381u9EBBmcKUJWbfn6QL82YvR
cUv81BVz1JWuA9uwECPvfsoyA1TqmFJBJaQUPYrxkdf44zQSiTZY
cVV1MBAgdAnrX8bjQWSJcgZAbVtprR8kgdfez858ejvXs3qZWqn2


Confirm? (y/n): 
Calculating transaction...

Final fee rate: 10.00 satoshis per vbyte

Raw signed transaction (hex):
02000000011d56acc4eee0baf9ae4e062d64ded32af701bbe0d3ebd8cb3686377f0c58718d02000000fd88010047304402205b215649efee3668ea4df5b38023f4232c9db9077588ef20b6d242858c8523f10220557bfe5b4345b9ec9773f7ce1729dfbdddc7a9987964c8ec8f7b932b5605623901473044022044f96125d54319e92e876965756b4c8188a979ea86424bef9c136c52466cf2a80220723179ff9e394f5ac354c8d2d109d302defe904a8c15a5a351b80e183ea68cfe0147304402203e55ded80ee250564b7a01326fb1e87694c0f59556fd55a0e15885a22cc242f30220382746d2220cebaae3aefdabe19eed3adae8ec8c01c559969b3260a2956d55de014cad53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155aeffffffff014a1a3101000000001976a914b6c7733e23a271e57ca16e5cd885174917968db988ac00000000

Transaction fingerprint (md5):
22c0b891bac7328decdaeaf7d5fab3cf
QR code for transaction written to transaction.png
